### ISSUE TITLE
no-unused-classes + descendant selectors

### ISSUE THREAD

**Issue by satazor** [atfzl/eslint-plugin-css-modules#22]
```css
// Hero.css
.hero {
    ...
}

.hero .icon-add {
    ...
}

.hero.small .icon-add {
   ...
}
```

CSS modules exports an object with this shape:

```js
{
  hero: '...',
  icon-add: '...',
  iconAdd: '...',
  small: '...'
}
```

And now some react component that uses it:

```js
// Hero.css
import styles from './Hero';

export default ({ small }) => (
   <div className={ classNames(styles.hero, { [styles.small]: small }) }>
      <i className={ styles.iconAdd }></i>
   <div>
);
```

As you see, all the styles are being used but `eslint-plugin-css-modules` reports that `icon-add` isn't used.

**Comment by satazor** [comment]
Ok I pinpointed this and the issue is that I have the camelCase option activated and `eslint-plugin-css-modules` doesn't detect them. Could we have an option, just like `css-loader` [has](https://github.com/webpack-contrib/css-loader#options), so that we support camelcase usage?

**Comment by satazor** [comment]
Duplicate of #6
